Cabal/src/Distribution/PackageDescription/Check.hs

```diff
@@ -999,14 +1006,14 @@ pd2gpd pd = gpd
 -- present in our .cabal file.
 checkMissingDocs
   :: Monad m
-  => [Glob] -- data-files globs.
-  -> [Glob] -- extra-source-files globs.
-  -> [Glob] -- extra-doc-files globs.
-  -> [Glob] -- extra-files globs.
+  => [Either FilePath Glob] -- data-files globs.
+  -> [Either FilePath Glob] -- extra-source-files globs.
+  -> [Either FilePath Glob] -- extra-doc-files globs.
+  -> [Either FilePath Glob] -- extra-files globs.
   -> CheckM m ()
 checkMissingDocs dgs esgs edgs efgs = do
   extraDocSupport <- (>= CabalSpecV1_18) <$> asksCM ccSpecVersion
-
+  mpackageOps <- asksCM $ ciPackageOps . ccInterface
```

Comment:
this feels pretty awkward to me - `liftInt` doesn't give a way to use two interfaces, so I have to smuggle this one in here. Could push the `liftInt` calls down to using the appropriate interfaces more locally.